app/components/SkeletonLoader.tsx

```diff
@@ -0,0 +1,33 @@
+export const SkeletonLoader = () => {
```

Comment:
is this general?
the loaders are different between pages

Reply:
If we have the time i would like to provide different loaders for each page. 
currently it is the only one we have